sources.cfg

```diff
@@ -0,0 +1,58 @@
+develop = .
+sources = sources
+always-checkout = force
+auto-checkout +=
+    bda.plone.cart
+    bda.plone.checkout
+    bda.plone.discount
+    bda.plone.orders
+    bda.plone.payment
+    bda.plone.shipping
+    bdajax
+    bda.plone.ajax
+    souper.plone
+    yafowil.plone
+    collective.z3cform.datagridfield
+    collective.js.jqueryui
+    collective.js.datatables
+    Products.CMFPlone
+
+
+[sources]
+bda.plone.cart                   = git https://github.com/bluedynamics/bda.plone.cart.git branch=plone5
+bda.plone.checkout               = git https://github.com/bluedynamics/bda.plone.checkout.git branch=plone5
+bda.plone.discount               = git https://github.com/bluedynamics/bda.plone.discount.git branch=plone5
+bda.plone.orders                 = git https://github.com/bluedynamics/bda.plone.orders.git branch=plone5
+bda.plone.payment                = git https://github.com/bluedynamics/bda.plone.payment.git branch=plone5
+bda.plone.shipping               = git https://github.com/bluedynamics/bda.plone.shipping.git branch=plone5
+
+bdajax                           = git https://github.com/bluedynamics/bdajax.git
+bda.plone.ajax                   = git https://github.com/collective/bda.plone.ajax.git branch=plone5
+
+#node.ext.zodb                    = git https://github.com/bluedynamics/node.ext.zodb.git
+#souper                           = git https://github.com/bluedynamics/souper.git
+souper.plone                     = git https://github.com/bluedynamics/souper.plone.git
+
+#yafowil                          = git https://github.com/bluedynamics/yafowil.git
+yafowil.plone                    = git https://github.com/bluedynamics/yafowil.plone.git branch=plone5
+#yafowil.widget.array             = git https://github.com/bluedynamics/yafowil.widget.array.git
+#yafowil.widget.autocomplete      = git https://github.com/bluedynamics/yafowil.widget.autocomplete.git
+#yafowil.widget.datetime          = git https://github.com/bluedynamics/yafowil.widget.datetime.git
+#yafowil.yaml                     = git https://github.com/bluedynamics/yafowil.yaml.git
+
+#plone.api                        = git https://github.com/plone/plone.api.git
+#plone.app.users                  = git https://github.com/plone/plone.app.users.git
+#plone.app.vocabularies           = git https://github.com/plone/plone.app.vocabularies.git
+#plone.app.workflow               = git https://github.com/plone/plone.app.workflow.git  branch=2.1.x
+
+#plonetheme.barceloneta            = git https://github.com/plone/plonetheme.barceloneta.git
+#Products.CMFPlone                 = git https://github.com/plone/Products.CMFPlone.git
+
+collective.z3cform.datagridfield = git https://github.com/collective/collective.z3cform.datagridfield.git
+collective.js.jqueryui           = git https://github.com/collective/collective.js.jqueryui.git branch=plone5
+collective.js.datatables         = git https://github.com/collective/collective.js.datatables.git branch=plone5
```

Comment:
collective.js.jqueryui and collective.js.datatables are already merged back into master iirc